xml/deployment_cluster_administration.xml

```diff
@@ -374,7 +374,7 @@ roleRef:
    installation of PTFs by running:
   </para>
 
-<screen>transactional-update reboot ptf install</screen>
+<screen>transactional-update reboot ptf install <replaceable>rpm ... rpm</replaceable></screen>
```

Comment:
Is this really the file name (as in foo-1.2.3.rpm) or the package name (foo or foo-1.2.3)?

If it is the latter, please change this to &lt;package&gt;&lt;replaceable&gt;package ... package&lt;/replaceable&gt;&lt;/package&gt;

Reply:
needs to be `.rpm` format, so i guess it can stay as it is